### DISCUSSION TITLE
Google地圖怎麼用ＡＩ幫你判斷最佳路徑

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1108] [General]
不論我們去外地旅遊又或是在本地遊玩
都很常需要google map的幫助
因為他會在最有效率的情況下幫你分析怎麼走最好
將龐雜的五種交通方式綜合分析
我覺得這個技術實在太厲害了
所以查了資料分享給大家
https://technews.tw/2020/09/04/how-google-maps-navigation-works/

**Comment by ghost** [comment]
一個大數據結合ai的產物，google地圖能運用機器學習技術產生預測結果，幫助想我一樣路痴的找到一條正確的道路，是很方便的一個軟體。

**Reply by wu51000** [reply]
那身為一個路痴有google map還走到迷路是不是很不可取嗚嗚嗚
